cba_settings_userconfig/cba_settings.sqf

```diff
@@ -104,16 +104,16 @@ force acex_field_rations_timeWithoutWater = 2;
 force acex_field_rations_waterSourceActions = 2;
 
 // ACE Fire
-force ace_fire_dropWeapon = 1;
-force ace_fire_enabled = true;
-force ace_fire_enableFlare = false;
-ace_fire_enableScreams = true;
+force force ace_fire_dropWeapon = 1;
+force force ace_fire_enabled = true;
+force force ace_fire_enableFlare = false;
+force force ace_fire_enableScreams = true;
 
 // ACE Fortify
-force ace_fortify_markObjectsOnMap = 1;
-force ace_fortify_timeCostCoefficient = 1;
-force ace_fortify_timeMin = 1.5;
-acex_fortify_settingHint = 2;
+force force ace_fortify_markObjectsOnMap = 1;
+force force ace_fortify_timeCostCoefficient = 1;
+force force ace_fortify_timeMin = 1.5;
+force force acex_fortify_settingHint = 1;
```

Comment:
Shouldn't this be just a change of default, without `force force`? .. So that non-eng don't get the notification, but people can change this setting anyway?